hack/local-up-cluster.sh

```diff
@@ -216,18 +249,24 @@ function start_apiserver {
     if [[ -n "${RUNTIME_CONFIG}" ]]; then
       runtime_config="--runtime-config=${RUNTIME_CONFIG}"
     fi
-
     APISERVER_LOG=/tmp/kube-apiserver.log
-    sudo -E "${GO_OUT}/kube-apiserver" ${priv_arg} ${runtime_config}\
+
+    # We provide a token auth, so that kubelets can access the api-server
+    # through a service endpoint.  This allows for highly available api-server
+    # setups.
+    sudo -E "${GO_OUT}/kube-apiserver" ${priv_arg} ${runtime_config} \
       --v=${LOG_LEVEL} \
       --cert-dir="${CERT_DIR}" \
       --service-account-key-file="${SERVICE_ACCOUNT_KEY}" \
+      --authorization-mode=AlwaysAllow \
```

Comment:
Also, since local-up-cluster gets referenced for example invocations, if this does end up in here, we should use better examples of exposing the API... generate an ABAC file that grants the kubelet user permission, etc